scripts/synapse_sytest.sh

```diff
@@ -16,15 +16,26 @@ cd "$(dirname $0)/.."
 mkdir /work
 
 # PostgreSQL setup
-if [ -n "$MULTI_POSTGRES" ]; then
-    # In this mode we want to run synapse against multiple split out databases.
-
+if [ -n "$MULTI_POSTGRES" ] || [ -n "$POSTGRES" ]; then
     # We increase the max connections as we have more databases.
-    echo -e "max_connections=1000" >> /var/run/postgresql/data/postgresql.conf
+    sed -i -r "s/^max_connections.*$/max_connections = 500/" /var/run/postgresql/data/postgresql.conf
+
+    echo -e "fsync = off" >> /var/run/postgresql/data/postgresql.conf
+    echo -e "full_page_writes = off" >> /var/run/postgresql/data/postgresql.conf
 
     # Start the database
     su -c 'eatmydata /usr/lib/postgresql/*/bin/pg_ctl -w -D $PGDATA start' postgres
 
+    su -c psql postgres <<< "show config_file"
+    su -c psql postgres <<< "show max_connections"
```

Comment:
I realized we could `grep` for the expected values here if we wanted to enforce failure if the configuration isn't as expected. Something like `su -c grep 500 $(psgl postgres <<< "show max_connections")`...not 100% sure that'll work with the `su` though.

Reply:
Yup, we could, but I think just printing the values for now is Good Enough. If it breaks again at least we will have some debugging info